diff --git a/node_modules/@types/roslib/index.d.ts b/node_modules/@types/roslib/index.d.ts
index 6099ab4..9c8fc3b 100644
--- a/node_modules/@types/roslib/index.d.ts
+++ b/node_modules/@types/roslib/index.d.ts
@@ -491,7 +491,9 @@ declare namespace ROSLIB {
          * @param {function} callback - Function with the following params:
          * @param {TMessage} callback.message - The published message.
          */
-        subscribe(callback: (message: TMessage) => void): void;
+        //OVERRIDE
+            //    subscribe(callback: (message: TMessage) => void): void;
+        subscribe<T>(callback: (message: {data : T}) => void): void;
 
         /**
          * Unregister as a subscriber for the topic. Unsubscribing will stop
